dashboard/app/models/levels/level.rb

```diff
@@ -855,6 +855,9 @@ def summarize_for_lab2_properties(script, script_level = nil, current_user = nil
     properties_camelized[:usesProjects] = try(:is_project_level) || channel_backed?
     properties_camelized[:finishUrl] = script_level.next_level_or_redirect_path_for_user(current_user) if script_level
     properties_camelized[:baseAssetUrl] = Blockly.base_url
+    properties_camelized[:hasValidation] = validated?
```

Comment:
Do we need this given we already send `validations` as a field, and we are converting the data on the frontend anyway? Could we just check for the existence of `validations` on the frontend?

Reply:
I wondered about this. Updated on the front end!